linalg/hypre_parcsr.hpp

```diff
@@ -45,7 +45,14 @@ void hypre_ParCSRMatrixEliminateAXB(hypre_ParCSRMatrix *A,
 void hypre_ParCSRMatrixEliminateAAe(hypre_ParCSRMatrix *A,
                                     hypre_ParCSRMatrix **Ae,
                                     HYPRE_Int num_rowscols_to_elim,
-                                    HYPRE_Int *rowscols_to_elim);
+                                    HYPRE_Int *rowscols_to_elim,
+                                    int ignore_rows = 0);
+
+/** Eliminate rows from a hypre ParCSRMatrix, setting all entries in the listed
+    rows of the matrix to zero. */
+void hypre_ParCSRMatrixEliminateRows(hypre_ParCSRMatrix *A,
```

Comment:
This goes beyond this PR, but has anyone looked at upstreaming these functions to HYPRE? Any thoughts @tzanio ?

Reply:
This is on my todo list...